CHANGELOG.adoc

```diff
@@ -55,6 +55,7 @@ In the near future, the GraphQL schema and the frontend of `sirius-components-fo
 - https://github.com/eclipse-sirius/sirius-web/issues/2371[#2371] [sirius-web] Default colors are now loaded from a default model located in `sirius-components-view/src/main/resources/studioColorPalettes.json`.
 They are loaded in every studio project but hidden in the explorer.
 They are also made readonly in the details view.
+- https://github.com/eclipse-sirius/sirius-web/issues/2388[#2388] [tree] `treeItem` is no longer composed of a single string for the imageURL, but of a list of strings to take into account composite images.
```

Comment:
Document the change in `IObjectService` as an API break since all our consumers who are using EMF data in a different configuration or who are not using EMF data at all are using a custom `ObjectService` instance

Reply:
Done.